components/automate-ui/src/app/entities/cds/cds.actions.ts

```diff
@@ -9,7 +9,10 @@ export enum CdsActionTypes {
   GET_CONTENT_ITEMS_FAILURE      = 'CDS::GET::CONTENT_ITEMS::FAILURE',
   INSTALL_CONTENT_ITEM           = 'CDS::INSTALL::CONTENT_ITEM',
   INSTALL_CONTENT_ITEM_SUCCESS   = 'CDS::INSTALL::CONTENT_ITEM::SUCCESS',
-  INSTALL_CONTENT_ITEM_FAILURE   = 'CDS::INSTALL::CONTENT_ITEM::FAILURE'
+  INSTALL_CONTENT_ITEM_FAILURE   = 'CDS::INSTALL::CONTENT_ITEM::FAILURE',
```

Comment:
For all the UI code below. The file is downloaded in the effects file. For an error or success, a message is shown in the banner.